src/validation.js

```diff
@@ -0,0 +1,30 @@
+const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const isValideUrl = (text, toCase) => {
+  const errors = [];
+
+  if (!text) {
+    errors.push({
+      message: 'Text to convert is required. '
```

Comment:
It's a bad way to represent string. If you have trouble with linter, you can add eslint-disable rule or change eslint rule.